addons/mail/models/mail_thread.py

```diff
@@ -2498,7 +2498,9 @@ def _notify_thread_by_email(self, message, recipients_data, msg_vals=False,
             return True
```

Comment:
Can you fixup the 3rd commit into the 1st one? I don't think it warrants two commits ^^
And move the CLA commit to be the first while you're at it? Just seems to make more sense

Also make sure your commit message lines don't go past 72 characters, and 50 characters for the title. For long urls you can use this syntax

```
see [this url][1] for more information

[1]: https://[...]
```
And it will create a link in the commit without needing the link straight in the middle of the message